internal/metamorph/callbacker.go

```diff
@@ -124,9 +124,14 @@ func (p *Callbacker) SendCallback(logger *slog.Logger, tx *store.StoreData) {
 		status.ExtraInfo = &tx.RejectReason
 	}
 
-	for i := 0; i < CallbackTries; i++ {
+	for _, callback := range tx.Callbacks {
+		p.sendCallback(logger, tx, callback, status, sleepDuration)
```

Comment:
Maybe this could be run in a goroutine, so we send all callbacks at the same time instead of one-by-one

Reply:
I think it is better to send them one by one.

That seems to be a problem with the way we send callbacks currently. As we send them in parallel we can overwhelm the callback endpoints. That can lead to failing callbacks even though the callback endpoints exist.